### PR TITLE
[W-10187060] Release Gating Card for Spring 22

### PR DESCRIPTION
**Pull request by kg345**
# Changes

- Spring '22 card will now appear under Release Management in Education Cloud Settings. 

# Issues Closed
[W-10187060
](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000PnDSRYA3/view)

# New Metadata

* Labels
   * stgReleaseGateEDASpring22Label
   * stgReleaseGateEDASpring22Description
   * stgReleaseGateEDASpring22FeatureDescription
   * stgReleaseGateEDASpring22FeatureHelpLabel

* Apex Class
   * EDASpring22ReleaseGate.cls
   * EDASpring22ReleaseGate_TEST.cls  

# Testing Notes

1. Go to Education Cloud Settings tab
2. Click on release management
3. Under "Education Data Architecture", it should have a card with the following details

- Card header should say "Spring '22"
- Description header should say, "There were no features gated for this release therefore all features were automatically released with Spring'22"
- Bullet item 1 header should say, "Please see EDA's release notes for Spring'22 for full details.
- Bullet item 2 header should say, "Activate Spring'22"
- EDA release gating card should direct users to https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHOcSAM?tab=discussion&sort=LAST_MODIFIED_DATE_DESC

### CODE REVIEW

**Comment by kg345** [comment]
@aarsvoboda I ended up creating new labels for Spring 22 and just matching the naming convention from Winter '22. Looking at it more closely, I presume we could probably just rename the generic text under one label instead of separate ones for releases (`FeatureHelpLabel` = Follow EDA Releases in the EDA Release Announcements group)

For the redirect link, it currently just goes to the trailblazer group based on the WI although that won't necessarily take them easily to the release notes. I have an open question with Jason about whether only the latest release card should show or if both should show. If both show, I figure we should update the Winter '22 release notes redirect to the [trailblazer post](https://trailhead.salesforce.com/trailblazer-community/feed/0D54S00000BuNlcSAF) of the release notes?

**Review by aarsvoboda — APPROVED** [review]

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 1]
```diff
@@ -3951,6 +3951,22 @@
...
+        <shortDescription>EDA Spring 22 release gate label</shortDescription>
+        <value>Spring&apos;22</value>
+    </labels>    
+    <labels>
+        <fullName>stgReleaseGateEDASpring22Description</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>EDA Spring 22 release gate description</shortDescription>
+        <value>There were no features gated for this release therefore all features were automatically released with Spring&apos;22</value>
```
Comment (ZeeshanSyedH) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:3968]: - [Future Enhancement] Parameterize release name to mitigate custom labels introduced per release.

Comment (ZeeshanSyedH) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:3968]: - [Future Enhancement] Parameterize release name to mitigate custom labels introduced per release.

**Review by ZeeshanSyedH — APPROVED** [review]
Looks good @kg345!

**Pull request merged**